### PR TITLE
Add section on server localization instructions

### PR DESCRIPTION
**Pull request by prabhanshuguptagit**
This accompanies the transifex config changes in https://github.com/simpledotorg/simple-server/pull/995.

### CODE REVIEW

File: engineering/localization/server.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+---
+description: Specific information related to localizing the rails app
+---
+
+# Server
+
+The backend uses Transifex to manage translations, similar to the android workflow.
```
Comment (harimohanraj89) [inline engineering/localization/server.md]: Let's hyperlink "android" if possible. Also, capital Android?

File: engineering/localization/server.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+The locale files sit in [`config/locales`](https://github.com/simpledotorg/simple-server/tree/master/config/locales).
+
+On the transifex dashboard, click on `simple-server` > `Resources` to look at the source files.
+
+### Language mappings
+
+In the GitHub integration settings, there is a language mapping section that tells Transifex how to convert from the
```
Comment (harimohanraj89) [inline engineering/localization/server.md]: These settings are in Transifex, yes? Can we make that explicit?

File: engineering/localization/server.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+```yaml
+settings:
+  language_mapping:
+    pa: pa_Guru_IN
+    mr: mr_IN
+    om: om_ET
+    ti: ti_ET
+```
+
+On the left are locale codes in the Transifex convention, on the right are locales in the convention that rails expects.
```
Comment (harimohanraj89) [inline engineering/localization/server.md]: rails -> Rails

**Review by harimohanraj89 — APPROVED** [review]
Tiny requests. Looks great.

File: engineering/localization/server.md (hunk 3 above)
Comment (prabhanshuguptagit) [reply engineering/localization/server.md]: 👾

**Pull request merged**